### ISSUE TITLE
Better camera quality or a plugin instead of WebcamTexture

### ISSUE THREAD

**Issue by cancaglar** [asus4/tf-lite-unity-sample#323]
Is there a way to improve FPS and reduce lag in Unity applications made with MoveNet or Blazepose when using 1920x1080 or 4k cameras on Windows? Alternatively, on mobile platforms, is there a way to use an external plugin to obtain higher quality view instead of Unity's WebcamTexture, or is it possible to implement such a solution?

**Comment by asus4** [comment]
@cancaglar Yes, the latency will improve if you use other low-latency camera plugins, such as [AVPro Live Camera](https://assetstore.unity.com/packages/tools/video/avpro-live-camera-3683) or [NatML VideoKit](https://github.com/natmlx/videokit). 

However, the camera resolution of the model (not camera preview) would be difficult to improve since pre-trained models accept fixed-sized images. The camera input is trimmed to match this size when you call the model via BaseImagePredictoer.

- [BlazePose - 224 x 224](https://developers.google.com/mediapipe/solutions/vision/pose_landmarker)
- [MoveNet - 256 x 256](https://tfhub.dev/google/movenet/multipose/lightning/1)
